<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmployeForm</class>
 <widget class="QWidget" name="EmployeForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLineEdit" name="leSearch">
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Rechercher</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="pbSearch">
     <property name="text">
      <string>Chercher</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" rowspan="3">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="3" rowspan="2" colspan="3">
    <widget class="QWidget" name="widget" native="true"/>
   </item>
   <item row="1" column="0" rowspan="2" colspan="2">
    <widget class="QListWidget" name="lwEmployes"/>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="pbReturnClear">
     <property name="text">
      <string>Retour</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QPushButton" name="pbClearSupp">
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
   </item>
   <item row="2" column="5">
    <widget class="QPushButton" name="pbAjoutMod">
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
